modules/ROOT/pages/dataweave-cookbook-conditional-list-reduction-via-function.adoc

```diff
@@ -4,12 +4,14 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: studio, anypoint, transform, transformer, format, aggregate, rename, split, filter convert, xml, json, csv, pojo, java object, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping
 
-//TODO: EXAMPLE NOT WORKING, CHANGED when to if.
 
-This DataWeave example flattens the input to something simpler. For each element in the input, it conditionally includes a "contentTypes" field only in case it has any values. It then rearranges this content into a more readable shape.
+This DataWeave example flattens the input to something simpler. For each element
+in the input, it conditionally includes a "contentTypes" field only in case it
```

Comment:
"contentTypes" > `contentTypes` In the following coming doc dataweave-cookbook-define-function-to-flatten-list.adoc you called contentTypes with backtips

Reply:
done